README.md

```diff
@@ -28,19 +28,28 @@ HTML and CSS
 3.html & css
 4.github
 
-### Live Demo
+#### Live Demo
 
-**Link:** <http://127.0.0.1:5500/index.html>
+This site was built using [GitHub Pages](http://127.0.0.1:5500/index.html/).
```

Comment:
- [ ] It is good you provided a live demo for your project, but it is broken. Providing links that lead no where plays on the professionalism of the repo. Since providing a live demo link is not an obligatory requirement for this milestone, consider removing it or providing a working one. You can used [githack](https://raw.githack.com/). Simply copy your [`index.html` path](https://raw.githack.com/ismailakinkunmi/portfolio-site/middle_section/index.html) and paste it in githack and it will generate your [live demo link](https://raw.githack.com/ismailakinkunmi/portfolio-site/middle_section/index.html) . You can simply use the path I have generated.